### ISSUE TITLE
Display Issues using Dark Mode

### ISSUE THREAD

**Issue by jmrowlands1** [microsoft/vsts-extension-retrospectives#409]
1. When Items are grouped and you expand the card the expanded cards all display white with white text.
![Capture1](https://user-images.githubusercontent.com/109124446/182286188-4a6ec973-0494-41ac-bc10-d26c3f52220b.PNG)

3. When in focus mode some label text is displayed in vary dark grey not white (# Items, Timer & Card Number). Also if you expand the card the grouped card text display in vary dark grey
![Capture2](https://user-images.githubusercontent.com/109124446/182286215-b19e0d9b-7f61-473c-b510-d824f4e6ed3a.PNG)

**Comment by jyang81** [comment]
I have the same issue

**Comment by mindlessroman** [comment]
Working on a fix!

**Comment by mindlessroman** [comment]
@jyang81 - I believe this should be resolved with the latest release (1.80.13). Can you confirm?

**Comment by jmrowlands1** [comment]
Yes looking a lot better - thanks

Jonathan

**Comment by mindlessroman** [comment]
Awesome! I'll close this out for now, and we can open a new issue if something comes up!
